.github/workflows/dotnet-publish.yml

```
@@ -8,45 +8,44 @@ on:
-      run: dotnet publish src/YoloKonsole/YoloKonsole.csproj -c Release -o releases/win-x64 --os win
-    - name: Create Release
-      id: create_release
-      uses: actions/create-release@v1
-      env:
-        GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }} # This token is provided by Actions, you do not need to create your own token
-      with:
-        tag_name: ${{ github.ref }}
-        release_name: YoloKonsole ${{ github.ref }}
-        body: |
-          ## HOWTO (win-x64):
+      - uses: actions/checkout@v2
```

actionlint (1.7.7)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)